### PR TITLE
Add extra (optional) parameters for joints

### PR DESCRIPTION
**Pull request by pdenes**
This allows to set "Profile_Velocity", "Profile_Acceleration" and PID values for individual joints.

### CODE REVIEW

File: dynamixel_hardware/src/dynamixel_hardware.cpp [hunk 1]
```diff
@@ -38,6 +38,15 @@ constexpr const char * kPresentVelocityItem = "Present_Velocity";
 constexpr const char * kPresentSpeedItem = "Present_Speed";
 constexpr const char * kPresentCurrentItem = "Present_Current";
 constexpr const char * kPresentLoadItem = "Present_Load";
+constexpr const char* const extraJointParameters[] = {
```
Comment (youtalk) [inline dynamixel_hardware/src/dynamixel_hardware.cpp:41]: Please follow the Google Coding Rule.

```suggestion
constexpr const char* const KExtraJointParameters[] = {
```

**Review by youtalk — COMMENTED** [review]
Thank you for your improvement.
I have a minor suggestion.

File: dynamixel_hardware/src/dynamixel_hardware.cpp (hunk 1 above)
Comment (pdenes) [reply dynamixel_hardware/src/dynamixel_hardware.cpp]: Sure! :+1: I've updated my PR now.

**Comment by youtalk** [comment]
Thank you!

**Pull request merged**